feature/stn/aggregate/index.js

```diff
@@ -115,9 +115,18 @@ ee.on('feat-stn', function () {
   registerHandler('bstXhrAgg', storeXhrAgg)
   registerHandler('bstApi', storeSTN)
   registerHandler('errorAgg', storeErrorAgg)
+  registerHandler('pvtAdded', processPVT)
 })
 
-function storeTiming (_t) {
+function processPVT (name, value, attrs) {
+  var t = {}
+  var owner = name === 'load' ? 'window' : 'document'
+  t[name] = value
+  storeTiming(t, true, owner)
+  if (hasFID(name, attrs)) storeEvent({type: 'fid', target: owner}, owner, value, value + attrs.fid)
```

Comment:
Do we want to store FID as an event? The event is really the FI (first input), and FID is its attribute. I can't think of a different way without adding a field in the payload, but wanted to double check.